components/input/Input.tsx

```diff
@@ -144,6 +144,9 @@ class Input extends React.Component<InputProps, InputState> {
   }
 
   componentDidMount() {
+    this.setState({
+      focused: true,
```

Comment:
`focused` is controlled by `onFocus` & `onBlur` event. Should not modify here.

Reply:
I mean I have to make a change only on the index.TSX file?

Reply:
Yes. Just fix with the page code.

Reply:
I think the issue is somehow different basically it will initialize the page again onClick of "Expand All Code" This is the reason why it scrolls up the page.
So, What I suggest is to take one boolean flag and initialize it to true at the time of page initialization and set as false below. or you could have suggest a better solution.

Reply:
The Input component works as expect. Should check the reason why page re-render when expanded.

Reply:
icon index.md file use inject react which makes IconDisplay re-render.
`````
```__react
import IconDisplay from 'site/theme/template/IconDisplay';
ReactDOM.render(<IconDisplay />, mountNode);
```
`````

We may use sessionStorage to save the state that already rendered once.

Reply:
Not working with session storage I think we need to restrict the call on click of expand all button.